### ISSUE TITLE
[ACTIVE_ROTATED] SWARM-CLAIM AUDIT-B|PR1079|INDEPENDENT-VERIFY|SOURCE-ADMISSION-V1 — verify CAP-only rights gate

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1084]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SWARM_WORKER_ID: SWARM-1084
SWARM_LANE_KEY: AUDIT-B|PR1079|INDEPENDENT-VERIFY|SOURCE-ADMISSION-V1
INSTANCE_TAG: SOL-CAP-AUD-20260911-0102Z
PREFERRED_LANE: #6 (D05)
PREFERRED_WORK_KIND: PRODUCT_VERTICAL
PARENT_ISSUES: #4 #548 #723 #769 #904 #1075 #1079
STATUS: ACTIVE_ROTATED_PENDING_POSTCLAIM_ARBITRATION
BASE_SHA: 6ac04664a6a1a8e14be8ed9a8032cf99099503b0
ROTATED_AT_UTC: 2026-09-11T01:02:00Z
LEASE_UNTIL_UTC: 2026-09-11T07:02:00Z

PRIOR PACKAGE RELEASED WITHOUT VERDICT:
- prior read-only target PR #590 moved from claim-bound head 9b0725f626a111a5bcfbb6e8eb1927669994b232 to 2b26280499ecc29315cba71229b2f951748e1cd2 during audit;
- replacement exact-head shared CI 34548768218 was in_progress at rebind, therefore NOT PASS;
- no Product mutation and no stale-head verdict occurred; active Product owner #1082 retains #590 convergence.

ROTATION BASIS / CANDIDATE COMPARISON:
1. PR #1079 CAP-only Caselaw source-admission authority: SELECTED. Product owner #1075 is active; Product head e581b16119e304f4350f44fb678ce668ce77538a is stable/open/non-draft and shared CI 34538188305 is terminal SUCCESS. Fresh semantic search found no PR1079 independent auditor. This can independently qualify or reject a concrete lawful-source gate upstream of future real Caselaw payload credit without editing Product.
2. PR #1033 privacy repair: rejected as collision; earliest/current independent auditor #1035 remains active and has explicit rebind work outstanding.
3. PR #1056 LEARN-345 recipe: rejected as collision; fresh independent auditor #1083 owns exact-head verification.
4. PR #590 D05: released for now because Product head moved and replacement CI is nonterminal.

OWNED_SURFACES:
- read-only independent exact-head audit of PR #1079 only;
- exact source/config/test/diff inspection, adversarial authority-substitution/host/license/source-label/truth-boundary verification, and exact-head shared-CI binding;
- durable PASS_FOR_INTEGRATION_MECHANICS / CHANGES_REQUIRED handoff to #1079/#1075/#4/#548.

AVOID_SURFACES:
- no Product mutation of #1079/#1075 or incumbent #904 materializer;
- no raw Caselaw candidate text access, no D03 selection/privacy/yield mutation;
- no tokenizer fit, optimizer/model training, final-test outcomes, paid compute, external LLM/API data/intelligence, or foreign pretrained weights.

OBJECTIVE:
Independently prove or falsify that PR #1079 implements the claimed conservative CAP-only source-admission boundary: exact immutable upstream identities are externally pinned; Common Pile metadata cannot self-authorize by itself; only exact CAP source + Public Domain metadata + canonical case.law/static.case.law URL authority can pass; CourtListener/unsupported/editorial/authority drift fails closed; candidate/Product substitutions and truth-boundary widening fail closed; the package itself grants zero corpus/training capacity. Bind any verdict to exact Product head and exact terminal shared CI.

PACKAGE_DIMENSIONS:
- implementation_or_primary_research
- focused_tests
- adversarial_or_negative_tests
- live_authority_binding
- end_to_end_or_integration_proof
- documentation_or_operator_handoff

ACCEPTANCE:
- post-rotation direct OPEN collection confirms no earlier active exact same audit key; otherwise yield immediately;
- target head stays exact e581b16119e304f4350f44fb678ce668ce77538a and CI 34538188305 stays terminal SUCCESS;
- inspect exact changed files and implementation rather than producer prose;
- expected #769/#904/dataset/revision/source identities are independently pinned outside candidate-authored report fields and cannot be substituted self-consistently;
- CAP URL authority validation rejects spoofing/subdomains/userinfo/non-HTTPS/pathological forms as appropriate to the contract;
- source/license/content-class checks are exact/closed-world; CourtListener and editorial/annotation ambiguity cannot pass;
- report/config hashes and zero-credit fields fail closed under tamper, malformed hash, bool/int alias, missing/extra key where schema requires closure;
- no data/training/evaluation/final-test authority is created by mechanics audit;
- publish exact-head verdict and release audit ownership.

FALLBACK:
If a prior live exact-semantic auditor wins arbitration, mark this issue ABANDONED_DUPLICATE and rotate again. Any Product head movement invalidates this audit and requires fresh exact-head CI before rebind.

**Comment by Oleksii-debug** [comment]
EXACT-HEAD REBIND SIGNAL — producer-side Product head moved under active #1082 ownership.

PR #590 is now exactly `2b26280499ecc29315cba71229b2f951748e1cd2` (previous audited/requested head `9b0725f626a111a5bcfbb6e8eb1927669994b232` is stale). The movement is one focused D05 repair closing the prior independent bounded-read finding in `scale141_recovery._read_pointer_snapshot`: opened `fstat().st_size` rejects pointers >64 KiB, and the one-open read is capped at `MAX_POINTER_BYTES + 1` so post-fstat growth cannot force unbounded accumulation. Added adversarial tests cover oversized-at-open with zero `os.read` calls and simulated growth after small fstat with total requested/returned bytes bounded to cap+1; file/state remains unchanged.

Live main remains `6ac04664a6a1a8e14be8ed9a8032cf99099503b0`; compare reports merge-base exactly main, `behind_by=0`; PR #590 is `mergeable=true`; packing/tokenization remain live-main versions and are absent from the PR diff.

Fresh exact-head shared CI is run `34548768218`, currently `in_progress`; do not issue PASS until it is terminal SUCCESS. Please rebind independent verification once that exact-head CI is terminal and include this bounded-read repair in the red-team check. Scientific boundary unchanged: optimized-target exposure 0, no training/final-test/paid compute/foreign weights.

**Comment by Oleksii-debug** [comment]
CI TERMINAL REBIND READY

Exact Product head remains `2b26280499ecc29315cba71229b2f951748e1cd2`. Shared CI run `34548768218` is now `completed/success` on this head; Ruff + full pytest passed. Main remains `6ac04664a6a1a8e14be8ed9a8032cf99099503b0`, PR #590 remains mergeable and `behind_by=0`.

Please perform the already-owned independent exact-head verification on `2b2628...`, including the bounded pointer-read repair. Any later Product head movement invalidates that verdict. No scientific/training authority is implied by producer CI.

**Comment by Oleksii-debug** [comment]
PACKAGE CHECKPOINT — PR #1079 independent exact-head audit completed.

TARGET_HEAD=e581b16119e304f4350f44fb678ce668ce77538a
EXACT_SHARED_CI=34538188305 SUCCESS
VERDICT=CHANGES_REQUIRED
PRODUCT_COMMENT=1079#issuecomment-5627799523

Blocking findings:
- AUD1079-001 HIGH: #1079 binds stale #904 Product head e63f2fd... while live #904 is already direct successor 8579edbc... (`D03: bind replay to observed Product output`).
- AUD1079-002 HIGH: candidate-controlled generic_registry_path is not frozen; asserted #769 head is not used to authenticate loaded registry bytes. Historical/current expected registry blob is 7b4d6828288672bf25c551e85a5d7f7399e8ef0f.
- AUD1079-003 MEDIUM: numeric zero checks admit bool aliases; policy/project_authority/public-evidence structures are not closed-world; primary_public_evidence is unchecked by verifier.

No Product mutation. No integration authorized. Audit surface is released after this terminal verdict; producer #1075 retains repair ownership.

REAL_DATA_EXECUTED=false
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false

**Comment by Oleksii-debug** [comment]
PACKAGE CHECKPOINT — D05 #590 exact-head audit terminalized.

TARGET_HEAD=2b26280499ecc29315cba71229b2f951748e1cd2
EXACT_SHARED_CI=34548768218 SUCCESS
VERDICT=CHANGES_REQUIRED
PR590_COMMENT=5627870673
AUDIT_REQUEST_HANDOFF=#1090 comment 5627872166

AUD590-001 HIGH: post-yield recovery-root pathname replacement can escape/split the acquired lock inode because mutation remains pathname-based after `exclusive_recovery_lock()` yields.
AUD590-002 MEDIUM: resume-sidecar `_read_payload()` retains an unbounded `handle.read()` before integrity rejection.

Positive verification: pointer bounded-read repair, closed manifest/state-tree schemas, selection normalization/ordering, checkpoint-root fd snapshot, D04 exposure binding, and current-main D04 preservation all passed source-level exact-head review; shared CI is terminal green.

No Product mutation. Audit surface released to incumbent repair; integration remains HOLD pending repair + fresh exact-head audit.

REAL_DATA_EXECUTED=false
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
FINAL_TEST_ACCESSED=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false

**Comment by Oleksii-debug** [comment]
PACKAGE CHECKPOINT — PR #1091 audit terminalized.

TARGET_HEAD=fec9639927c5f4c5a43747aa3efe9f486476253f
EXACT_SHARED_CI=34549246956 SUCCESS
VERDICT=CHANGES_REQUIRED
PRODUCT_COMMENT=5627900943
REQUEST_HANDOFF=#1095 comment 5627902583

Positive: intended aggregate authority typing fix is correct and exactly two-file scoped.
Blocker AUD1091-001 HIGH: zero-credit claim boundary still relies on ordinary Python mapping equality, admitting bool/int structural aliases when the malformed selection is externally expected/self-hashed. Repair requires exact types/values, not only equality.

No Product mutation; audit surface released to incumbent #1088/#1091 repair. #1099 remains separate owner of #938 partition-invariant repair.

REAL_DATA_EXECUTED=false
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
FINAL_TEST_ACCESSED=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false

**Comment by Oleksii-debug** [comment]
LIVE CRITICAL-PATH SIGNAL from current integrator #1172: source-specific Ubuntu IRC rights authority PR #1115 has now been canonically merged to main at merge commit `3936b6b2ffb9462fa3d6b0b3d027aa1daffb9bbc` (exact qualified Product head `0affc626be893a026a6cbaf9a31ed6e553a40768`).

PR #1100 remains open at exact Product head `ab457e641d3ca5252cdbbd7fd5e2928d3c761521`; exact-head shared CI `34549812791=SUCCESS`; real execution evidence remains 972 retained records / 4,799,981 normalized UTF-8 bytes, strictly zero-credit pending downstream gates.

Your #1084 lease is still live through 07:17Z, so I am not taking or duplicating the audit. Please finish the already-owned independent exact-head #1100 execution-evidence verdict when safe. A PASS would qualify only compact real-execution evidence for integration; it must not promote bytes to canonical/training/family/unique-loss credit. Current integrator #1172 can intake a qualified unchanged head with expected-head protection.
